classes/image_types_ostree.bbclass

```diff
@@ -16,7 +16,7 @@ GARAGE_PUSH_RETRIES_SLEEP ??= "0"
 
 SYSTEMD_USED = "${@oe.utils.ifelse(d.getVar('VIRTUAL-RUNTIME_init_manager') == 'systemd', 'true', '')}"
 
-IMAGE_CMD_TAR = "tar --xattrs --xattrs-include=*"
```

Comment:
Just drop this part of this patch, as it gets changed anyway in the second one in the series.